### PR TITLE
clarifys that the nuclear bomb is nanotrasen branded

### PR DESCRIPTION
**Pull request by boodaliboo**
image

#### Changelog

:cl:  
rscadd: Added new sentence
/:cl:

### CODE REVIEW

**Comment by Reed0506** [comment]
DEEPEST LORE

**Comment by Reed0506** [comment]
should probly steal some description from the beer nukes

**Comment by Georgeur** [comment]
IN OUR DARKEST HOUR, THE LORE TEAM STOOD UP FOR US

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/nuclearbomb
 	name = "nuclear fission explosive"
-	desc = "You probably shouldn't stick around to see if this is armed."
+	desc = "A NanoTrasen branded nuclear bomb, You probably shouldn't stick around to see if this is armed."
```
Comment (LoliconSlayer) [inline code/modules/antagonists/nukeop/equipment/nuclearbomb.dm]: Specify that it's a stolen bomb and use a period instead of a comma because grammar

Comment (boodaliboo) [reply code/modules/antagonists/nukeop/equipment/nuclearbomb.dm]: ok

**Comment by LoliconSlayer** [comment]
Lore team approved 👍

**Comment by ynot01** [comment]
its Nanotrasen not NanoTrasen

**Comment by boodaliboo** [comment]
> its Nanotrasen not NanoTrasen

no

**Review by JamieD1 — APPROVED** [review]
For god sake

**Pull request merged**